### ISSUE TITLE
E701 on type annotation of statement

### ISSUE THREAD

**Issue by pLOPeGG** [PyCQA/pycodestyle#873]
On python 3.6.8 and 3.7.3, pep8 outputs E701 when using type annotation on statements. I'm using conda on windows to install pep8 and pycodestyle.
This problem was fixed by #612 but still happens in my code. My variables don't start with keywords.

```
conda install pep8 pycodestyle
```

In VSCode when selecting pep8 as linter the following code is flagged by pep8:
```python
my_variable: int = 3
```
![image](https://user-images.githubusercontent.com/23520944/60504162-5cbca980-9cc1-11e9-8f1f-eb838c7c44bf.png)

This is not happening with flake8.

versions of libraries:
```
pep8: 1.7.1
pycodestyle: 2.5.0
flake8: 3.7.7
```

**Comment by FichteFoll** [comment]
pep8 has been deprecated long ago and pycodestyle is its successor (primarily a name change). pep8 isn't updated anymore.

**Comment by pLOPeGG** [comment]
Okay thought VSCode `pep8` option was automatically picking up pycodestyle instead of pep8 (it's implicit in the doc). Changing VSCode settings to pick pycodestyle instead of pep8 works by modifying `"python.linting.pep8Path"` to `pycodestyle
